doc/ci-cd/deployment.md

```diff
@@ -78,12 +79,11 @@ Checkout the tag you want to start from.
 #### Hard deploy (when MEP wanted, manual)
 
 - Know which version (and then tag) you want to deploy. If you patched a version Staging, it could be `patch/v1.X.Y`
-
 - `yarn trigger:production:deploy <tag>`
 
 This will create a tag `prod-hard-deploy`. CircleCI will detect the tag and launch the lane `deploy-android-production-hard` & `deploy-ios-production-hard` (see `.circleci/config.yml` file)
 
-### Hotfix
+### Hotfix (for production see 'Patch staging with additional commits' for staging)
```

Comment:
C'est pas "for Staging" ?

Ne faut-il pas surpprimer les lignes 101 et 102 ?

Reply:
statging n'est pas un nom propre donc je ne vois pas pourquoi mettre une maj ? 
Pour les lignes 101 et 102 surtout pas elles font partie du process de test avant d'envoyer en prod